### DISCUSSION TITLE
No translation key in Weblate

### DISCUSSION THREAD

**Discussion by antypo** [immich-app/immich#14557] [Q&A] [answered]
Hello,
I found a typo while using the app.
action_common_clear in Polish should be "Wyczyść".
I couldn't find it in Weblate, am I doing something wrong or is there a part of translations that's missing from there?
Thanks

**Comment by bo0tzz** [comment, marked as answer]
The mobile app isn't translated via Weblate yet, it's still in Localizely.

**Reply by antypo** [reply]
Thanks, could someone update it then? @bo0tzz
